tools/packaging/kernel/configs/fragments/gpu/nvidia.arm64.conf.in

```diff
@@ -16,7 +16,7 @@ CONFIG_MEMORY_FAILURE=y
 
 # VFIO/IOMMU setttings 
 CONFIG_MMU_NOTIFIER=y
-CONFIG_IOASID=y
```

Comment:
So, you need to also bump the kata kernel version file.

If this is used somewhere else, you may want to add it to the allow list we have.

Reply:
Argh.... always forget that. Done.